uint8 channel   # Channel that maps to a pin
uint16 value    # Value to set: duty cycle if analog, 1 or 0 if digital